packages/components/src/components/money/money.tsx

```diff
@@ -1,15 +1,30 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { formatMoney, getCurrencyDisplayCode } from '@deriv/shared';
 
-const Money = ({ amount, className, currency = 'USD', has_sign, should_format = true, show_currency = false }) => {
+type TMoneyProps = {
+    amount?: number | string;
+    className?: string;
+    currency?: string;
+    has_sign?: boolean;
+    should_format?: boolean;
+    show_currency?: boolean; // if true, append currency symbol
+};
```

Comment:
```suggestion
type TMoneyProps = {
    amount: number | string;
    className: string;
    currency: string;
    has_sign: boolean;
    should_format: boolean;
    show_currency: boolean; // if true, append currency symbol
};
```

Reply:
@yauheni-kryzhyk-deriv, many of the `components` that use this `component` don't pass in every prop. I checked `account-has-balance` for example and it doesn't pass in `className`  can you help to confirm this as well?
`payment-agent-withdraw-confirm` would be another

Reply:
@jim-deriv I just proposed to to use utility type Partial.
when all props are optional we can use Partial< >
https://www.typescriptlang.org/docs/handbook/utility-types.html